client/components/EventList.js

```diff
@@ -1,17 +1,29 @@
 const React = require('react')
 const store = require('../store')
+const moment = require('moment')
 
 const EventList = (props) => {
   const events = store.getState().events
-  const { name, date, time, notes } = props
+  const { name, day, time, notes } = props
+
+  events.sort((a, b) => {
+    if (a.date > b.date) {
+      return 1
+    }
+    if (a.date < b.date) {
+      return -1
+    }
+    return 0
+  })
```

Comment:
nice job here! quick note, `sort` actually modifies the array, you probably want to create a new array here, and sort that. You can use `Array.from(...)`

```js
const sortedEvents = Array.from(events).sort(...)
```